src/pages/ProblemEditor/Dialogs/SuccessDialog.tsx

```diff
@@ -0,0 +1,29 @@
+import { Button, Dialog, DialogActions, DialogTitle } from "@mui/material";
+import React from "react";
+import { useHistory } from "react-router";
+import { useAppSelector } from "../../../app/common/hooks";
+
+interface Props {}
+
+export const SuccessDialog = (props: Props) => {
+  const showSuccessModal = useAppSelector(
+    (state) => state.problemEditorContainer.showSuccessModal
+  );
+  const history = useHistory();
+  return (
+    <Dialog open={showSuccessModal} aria-labelledby="alert-dialog-title">
+      <DialogTitle id="alert-dialog-title">
+        Problem added successfully!
+      </DialogTitle>
+      <DialogActions>
+        <Button
+          onClick={() => history.push("/admin/modules")}
+          variant="contained"
+          color="success"
+        >
+          Ok!
+        </Button>
+      </DialogActions>
+    </Dialog>
```

Comment:
Since there are a couple dialogs, consider using a generic dialog component in a future PR, it is fine for this one for me. 
I created a generic dialog component and put it in the shared folder, feel free to try it and see if it fits your uses. 

You can also modify it. You might need to make `maxWidth` an optional argument in the dialog for your use case. 
You can pass buttons to the dialog like we do for the content container that you created.

I borrowed what you did for that bit.

Reply:
Thanks!  I definitely will be having a reusable dialog as part of the edit problem PR I'm working on